ui/src/app/utils/zendesk.ts

```diff
@@ -78,8 +80,9 @@ export const supportUrls: ZendeskUrls = ((env) => {
       genomicExtraction: article('4403537387412'),
       gpus: article('4428626363668'),
       gettingStarted: category('360005884571'),
-      tableOfContents: category('360005884591'),
```

Comment:
shall we have article for preprod?

Reply:
I don't know what the fate of this preprod zendesk instance is. I would rather see it turned down I think.

Reply:
FYI I checked with support on this, they are still figuring it out. If they wind up repicating this article to preprod, I'll send another PR.